pymc3/stats.py

```diff
@@ -858,7 +861,7 @@ def quantiles(x, qlist=(2.5, 25, 50, 75, 97.5), transform=lambda x: x):
         return dict(zip(qlist, quants))
 
     except IndexError:
-        pm._log.warning("Too few elements for quantile calculation")
+        _log.warning("Too few elements for quantile calculation")
```

Comment:
Why do we catch this exception? We just return `None` here. Maybe this should be
`raise ValueError('Too few elements for quantile calculation')`?